cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/sriov-cni-daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-sriov-cni-ds-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: sriov-cni
+spec:
+  selector:
+    matchLabels:
+      name: sriov-cni
+  template:
+    metadata:
+      labels:
+        name: sriov-cni
+        tier: node
+        app: sriov-cni
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: kube-sriov-cni
+        image: nfvpe/sriov-cni
```

Comment:
This one is breathing, while the others are not
for example nfvpe/sriov-device-plugin:v3.3

worth to either let them both breath or freeze them both (i guess the 2nd is much better).
also best to put it on quay.io, else we hit rate limit, if no proxy is used (locally)
proxy with kind locally is not comfortable imo

Reply:
I pushed a change now both images names and tags controlled from `Kustomization` manifest.